scripts/archive-artifacts.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+#
+# Copyright 2014-2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# -------------- Import ---------------------#
+export BASE_DIR=${BASE_DIR:-"/opt/rpc-openstack"}
+source ${BASE_DIR}/scripts/functions.sh
+# ------------ End import -------------------#
+
+#---------- Required Vars -------------------#
+export BUILD_TAG=${BUILD_TAG:-""}
+export CONTAINER=${CONTAINER:-""}
+export SRC=${SRC:-""}
+#--------------- End Vars -------------------#
```

Comment:
I'm not sure I understand the point of these vars. It places an unnecessary complexity on the repo owner to have to do these things.

My suggestion would be to simply make the directory something like ${WORKSPACE}/logs - put everything in there, then tarball/compress into logs.tar.bz2

The gating system then takes that file, renames it or whatever need to be done and handles the upload. The only requirement for the repo owner is to have a logs.tar.bz2 in the workspace (or some well known directory which *all* repositories use).